### ISSUE TITLE
OroCRM: Installer Fails on Install Assets Steps

### ISSUE THREAD

**Issue by BogdanZuy** [orocrm/crm#117]
![screenshot_2](https://cloud.githubusercontent.com/assets/5727079/8076272/f300fd66-0f52-11e5-9687-20c5590ee2de.png)

<code>
[2015-06-10 09:48:28] Launching "oro:migration:load" command
Process migrations...
  > Oro\Bundle\MigrationBundle\Migration\CreateMigrationTableMigration
  > Oro\Bundle\OrganizationBundle\Migrations\Schema\OroOrganizationBundleInstaller
  > Oro\Bundle\SecurityBundle\Migrations\Schema\v1_0\OroSecurityBundle
  > Oro\Bundle\AttachmentBundle\Migrations\Schema\OroAttachmentBundleInstaller
  > Oro\Bundle\EmailBundle\Migrations\Schema\OroEmailBundleInstaller
  > Oro\Bundle\UserBundle\Migrations\Schema\OroUserBundleInstaller
  > Oro\Bundle\SSOBundle\Migrations\Schema\OroUserBundleInstaller
  > Oro\Bundle\EntityConfigBundle\Migrations\Schema\OroEntityConfigBundleInstaller
  > Oro\Bundle\EntityConfigBundle\Migrations\Schema\v1_3\OroEntityConfigBundle
  > Oro\Bundle\EntityConfigBundle\Migrations\Schema\v1_4\FixOptionSetObjects
  > Oro\Bundle\EntityConfigBundle\Migrations\Schema\v1_5\DropFieldConfig
  > Oro\Bundle\EntityExtendBundle\Migrations\Schema\OroEntityExtendBundleInstaller
  > Oro\Bundle\EntityExtendBundle\Migrations\Schema\v1_3\OroEntityExtendBundle
  > Oro\Bundle\IntegrationBundle\Migrations\Schema\OroIntegrationBundleInstaller
  > Oro\Bundle\ActivityBundle\Migrations\Schema\OroActivityBundleInstaller
  > Oro\Bundle\ActivityListBundle\Migrations\Schema\OroActivityListBundleInstaller
  > Oro\Bundle\AddressBundle\Migrations\Schema\OroAddressBundleInstaller
  > Oro\Bundle\BatchBundle\Migrations\Schema\OroBatchBundleInstaller
  > Oro\Bundle\CalendarBundle\Migrations\Schema\OroCalendarBundleInstaller
  > Oro\Bundle\CalendarBundle\Migrations\Schema\v1_9\OroCalendarBundle
  > Oro\Bundle\ConfigBundle\Migrations\Schema\OroConfigBundleInstaller
  > Oro\Bundle\CronBundle\Migrations\Schema\v1_0\JmsJob
  > Oro\Bundle\CronBundle\Migrations\Schema\v1_0\OroCronBundle
  > Oro\Bundle\CronBundle\Migrations\Schema\v1_1\JmsJob
  > Oro\Bundle\DataAuditBundle\Migrations\Schema\v1_0\OroDataAuditBundle
  > Oro\Bundle\DataAuditBundle\Migrations\Schema\v1_1\OroDataAuditBundle
  > Oro\Bundle\DataAuditBundle\Migrations\Schema\v1_2\OroDataAuditBundle
  > Oro\Bundle\DataGridBundle\Migrations\Schema\OroDataGridBundleInstaller
  > Oro\Bundle\EmbeddedFormBundle\Migrations\Schema\OroEmbeddedFormBundleInstaller
  > Oro\Bundle\ImapBundle\Migrations\Schema\v1_0\OroImapBundle
  > Oro\Bundle\ImapBundle\Migrations\Schema\v1_1\OroImapBundle
  > Oro\Bundle\ImapBundle\Migrations\Schema\v1_2\OroImapBundle
  > Oro\Bundle\InstallerBundle\Migrations\Schema\OroInstallerBundle
  > Oro\Bundle\MigrationBundle\Migrations\Schema\v1_0\OroMigrationBundle
  > Oro\Bundle\MigrationBundle\Migrations\Schema\v1_1\OroMigrationBundle
  > Oro\Bundle\NoteBundle\Migrations\Schema\v1_0\OroNoteBundle
  > Oro\Bundle\NoteBundle\Migrations\Schema\v1_1\OroNoteBundle
  > Oro\Bundle\NotificationBundle\Migrations\Schema\OroNotificationBundleInstaller
  > Oro\Bundle\ReportBundle\Migrations\Schema\OroReportBundleInstaller
  > Oro\Bundle\SearchBundle\Migrations\Schema\OroSearchBundleInstaller
    ERROR: Driver "%s" not found
  > Oro\Bundle\SegmentBundle\Migrations\Schema\OroSegmentBundleInstaller - skipped
  > Oro\Bundle\SidebarBundle\Migrations\Schema\v1_0\OroSidebarBundle - skipped
  > Oro\Bundle\SidebarBundle\Migrations\Schema\v1_1\OroSidebarBundle - skipped
  > Oro\Bundle\TagBundle\Migrations\Schema\OroTagBundleInstaller - skipped
  > Oro\Bundle\TranslationBundle\Migrations\Schema\OroTranslationBundleInstaller - skipped
  > Oro\Bundle\WindowsBundle\Migrations\Schema\v1_0\OroWindowsBundle - skipped
  > Oro\Bundle\WorkflowBundle\Migrations\Schema\OroWorkflowBundleInstaller - skipped
  > Oro\Bundle\CommentBundle\Migrations\Schema\v1_0\OroCommentBundle - skipped
  > Oro\Bundle\DashboardBundle\Migrations\Schema\OroDashboardBundleInstaller - skipped
  > Oro\Bundle\NavigationBundle\Migrations\Schema\v1_0\OroNavigationBundle - skipped
  > Oro\Bundle\NavigationBundle\Migrations\Schema\v1_1\OroNavigationBundle - skipped
  > Oro\Bundle\NavigationBundle\Migrations\Schema\v1_2\OroNavigationBundle - skipped
  > Oro\Bundle\NavigationBundle\Migrations\Schema\v1_3\OroNavigationBundle - skipped
  > OroCRM\Bundle\ContactBundle\Migrations\Schema\OroCRMContactBundleInstaller - skipped
  > OroCRM\Bundle\AccountBundle\Migrations\Schema\OroCRMAccountBundleInstaller - skipped
  > OroCRM\Bundle\MarketingListBundle\Migrations\Schema\OroCRMMarketingListBundleInstaller - skipped
  > OroCRM\Bundle\MarketingListBundle\Migrations\Schema\v1_2\OroCRMMarketingListBundle - skipped
  > Oro\Bundle\TrackingBundle\Migrations\Schema\OroTrackingBundleInstaller - skipped
  > Oro\Bundle\ReminderBundle\Migrations\Schema\OroReminderBundleBundleInstaller - skipped
  > OroCRM\Bundle\CallBundle\Migrations\Schema\OroCRMCallBundleInstaller - skipped
  > OroCRM\Bundle\CampaignBundle\Migrations\Schema\OroCRMCampaignBundleInstaller - skipped
  > OroCRM\Bundle\ChannelBundle\Migrations\Schema\OroCRMChannelBundleInstaller - skipped
  > OroCRM\Bundle\ChannelBundle\Migrations\Schema\v1_4\OroCRMChannelBundle - skipped
  > OroCRM\Bundle\TaskBundle\Migrations\Schema\OroCRMTaskBundleInstaller - skipped
  > OroCRM\Bundle\AnalyticsBundle\Migrations\Schema\v1_0\OroCRMAnalyticsBundle - skipped
  > OroCRM\Bundle\SalesBundle\Migrations\Schema\OroCRMSalesBundleInstaller - skipped
  > OroCRM\Bundle\ContactUsBundle\Migrations\Schema\OroCRMContactUsBundleInstaller - skipped
  > OroCRM\Bundle\MagentoBundle\Migrations\Schema\OroCRMMagentoBundleInstaller - skipped
  > OroCRM\Bundle\CaseBundle\Migrations\Schema\OroCRMCaseBundleInstaller - skipped
  > OroCRM\Bundle\CaseBundle\Migrations\Schema\v1_4\OroCRMCaseBundle - skipped
  > Oro\Bundle\MigrationBundle\Migration\UpdateBundleVersionMigration
  > Oro\Bundle\EntityExtendBundle\Migration\RefreshExtendCacheMigration - skipped
  > Oro\Bundle\EntityConfigBundle\Migration\UpdateEntityConfigMigration - skipped
  > Oro\Bundle\EntityExtendBundle\Migration\UpdateExtendConfigMigration - skipped
  > OroCRM\Bundle\ActivityContactBundle\Migration\ActivityContactMigration - skipped
  > Oro\Bundle\ActivityListBundle\Migration\ActivityListMigration - skipped
  > Oro\Bundle\EntityExtendBundle\Migration\UpdateExtendIndicesMigration - skipped

                                                                                          
  [RuntimeException]                                                                      
  Failed migrations: Oro\Bundle\SearchBundle\Migrations\Schema\OroSearchBundleInstaller. 
</code>

**Comment by BogdanZuy** [comment]
No worry. Working now.
- increased timeouts
- marked full drop db
- rechecked driver Mysql

**Comment by cljk** [comment]
I had a comparable problem.... I used *days* figuring out the problem. I always got "RuntimeException" on `OroWorkflowBundleInstaller`. 
Since I had a running system which was installed some weeks ago I could compare everything.
* PHP config - made everything the same
* composer.lock (since I only had the composer.json in VCS)
* the complete vendor-dir

Several reinstalls later I figured it out: It was the MySQL-Version(!)
I used a dockerized mysql-container on 8.0 and my prod was on 5.7. So I pulled the 5.7 image, deleted the database .... and it finally worked... without any error.

So there has to be a bug in association with MySQL 8. Since MySQL 8 is not released yet this is no issue of Oro but my own fault to use a not released version ... 
Such a dumb mistake on my side :-(

Finally I can start now implementing... :+1:
